robottelo/constants/__init__.py

```diff
@@ -2060,6 +2063,8 @@ class Colored(Box):
 EXPIRED_MANIFEST = 'expired-manifest.zip'
 
 # Data File Paths
+
+
```

Comment:
Were these lines intended to be added?

Reply:
my editor did this on save, made some config changes there :)